build.sbt

```diff
@@ -2,14 +2,15 @@ organization in ThisBuild := "ch.epfl.scala"
 
 version in ThisBuild := "0.2.0-SNAPSHOT"
 
-resolvers in ThisBuild += "scala-pr" at "https://scala-ci.typesafe.com/artifactory/scala-pr-validation-snapshots"
-scalaVersion in ThisBuild := "2.12.2-ebe1180-SNAPSHOT" // from https://github.com/scala/scala/pull/5742
+//resolvers in ThisBuild += "scala-pr" at "https://scala-ci.typesafe.com/artifactory/scala-pr-validation-snapshots"
+//scalaVersion in ThisBuild := "2.12.2-ebe1180-SNAPSHOT" // from https://github.com/scala/scala/pull/5742
+scalaVersion in ThisBuild := "2.12.1"
```

Comment:
Please drop this version change. https://github.com/scala/scala/pull/5742#issuecomment-282764184 actually says that bugfix is needed here:
> Rebased on top of 2.12.x head in order to build a version that also includes #5708, so I can use it in collection-strawman.

but dropping this version change from the PR is a good idea anyway.